From 0b8d867b0ea35173e1cd0e98915fe1492b269032 Mon Sep 17 00:00:00 2001
From: Antoine Martin <dev@ayakael.net>
Date: Sun, 6 Sep 2026 01:14:01 -0400
Subject: [PATCH] Support ghc-9.14 by widening a few constraints

This patch can be removed once following happens:
* yesodweb/persistent: addresses https://github.com/yesodweb/persistent/issues/1619
* haskell-party/feed: 1.3.2.2 is tagged, including https://github.com/haskell-party/feed/pull/78
---
 cabal.project | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/cabal.project b/cabal.project
index c4651e9..5d2cd8d 100644
--- a/cabal.project
+++ b/cabal.project
@@ -2,3 +2,8 @@ packages:
   .
 jobs: $ncpus
 semaphore: True
+
+allow-newer: persistent:template-haskell
+allow-newer: feed:base
+allow-newer: botan-low:base
+allow-newer: botan-bindings:base
-- 
2.54.0
